resources/ory/charts/hydra/charts/hydra-maester/templates/crd-job.yaml

```diff
@@ -15,6 +15,8 @@ spec:
         sidecar.istio.io/inject: "false"
     spec:
       serviceAccountName: {{ .Release.Name }}-crd-init
+      securityContext:
+        runAsUser: 2000
```

Comment:
Please make this configurable

Reply:
Yes, we can use variables instead. 
Will check and update this PR.